scripts/build.sh

```diff
@@ -33,7 +33,7 @@ $FM_BIN_DIR/configgen --out-dir $FM_CFG_DIR --num-nodes $FM_FED_SIZE --hbbft-bas
 export FM_LN1="lightning-cli --network regtest --lightning-dir=$FM_LN1_DIR"
 export FM_LN2="lightning-cli --network regtest --lightning-dir=$FM_LN2_DIR"
 export FM_BTC_CLIENT="bitcoin-cli -regtest -rpcuser=bitcoin -rpcpassword=bitcoin"
-export FM_MINT_CLIENT="$FM_BIN_DIR/mint-client-cli $FM_CFG_DIR"
```

Comment:
Is this a fix? Probably should make it another PR.

Reply:
I changed the `mint-client-cli` to not take the workdir as an argument but as a flag so other commands which dont need the workdir are not bothered by it